eth/tracers/internal/tracetest/util.go

```diff
@@ -46,6 +46,7 @@ func (c *callContext) toBlockContext(genesis *core.Genesis) vm.BlockContext {
 	}
 	if genesis.Config.IsLondon(context.BlockNumber) {
 		context.BaseFee = (*big.Int)(c.BaseFee)
+		context.Random = &genesis.Mixhash
```

Comment:
You might be right that the `Random` must not be `nil` ... but I don't see what `MixHash` has to do with it??

Reply:
fyi https://github.com/ethereum/execution-apis/issues/328

https://github.com/islishude/go-ethereum/blob/5b393ac85a26a6412db704368963d2b1470a76b0/miner/worker.go#L187-L188

https://github.com/islishude/go-ethereum/blob/5b393ac85a26a6412db704368963d2b1470a76b0/beacon/engine/types.go#L282

Reply:
It's hard to understand, but they're the same after the Merge

Perhaps geth should update the name